Neuro/Pathfinding/DataStructures/Node.cs

```diff
@@ -1,40 +1,19 @@
 ﻿using UnityEngine;
 
-namespace Neuro.Utils;
+namespace Neuro.Pathfinding.DataStructures;
 
 public class Node : IHeapItem<Node>
 {
     public bool accessible;
-    public Vector2 worldPosition;
-
-    public Node parent;
-
-    public int gridX;
-    public int gridY;
 
     public int gCost;
-    public int hCost;
-    public int fCost
-    {
-        get
-        {
-            return gCost + hCost;
-        }
-    }
 
-    int heapIndex;
+    public readonly int gridX;
+    public readonly int gridY;
+    public int hCost;
 
-    public int HeapIndex
-    {
-        get
-        {
-            return heapIndex;
-        }
-        set
-        {
-            heapIndex = value;
-        }
-    }
+    public Node parent;
+    public Vector2 worldPosition;
```

Comment:
public fields

Reply:
This whole class is a mess, I do not wish to modify it in this pull request.